main.go

```diff
@@ -125,6 +127,12 @@ func main() {
 		"Webhook server port (set to 0 to disable)",
 	)
 
+	flag.StringVar(&healthPort,
+		"health-port",
+		":9440",
+		"The address to which the health endpoint binds to.",
```

Comment:
If we want to match the description for metrics-addr, we should do "The address the health endpoint binds to." If we want to be grammatically correct, then "The address to which the health endpoint binds." I'd vote for consistency (i.e. the former), and we can address the grammar issue later if desired.

Reply:
updated